tests/InputFiles/ExampleExecutables/RandomAmr1D.yaml

```diff
@@ -3,10 +3,18 @@
 
 Executable: RandomAmr1D
 Testing:
-  Check: parse;execute
+  Check: parse;execute_check_output
+ExpectedOutput:
+ - SpectreCheckpoint000000
+Timeout: 10
 
 ---
 
+Criteria:
```

Comment:
Not for this PR, but now that I see this in an input file, I think the yaml group should be called `AmrCriteria`. I think that would be clearer.

Reply:
I think just add an `Amr` option group and put this tag in there.

Reply:
okay, I'll do that in my next AMR PR